lib/scoped/update.js

```diff
@@ -2,10 +2,45 @@ module.exports = update
 
 var find = require('./find')
 
+var merge = require('lodash/merge')
+var toId = require('pouchdb-hoodie-api/utils/to-id')
+
 function update (type, api, objectsOrIds, change) {
   return find(type, api, objectsOrIds)
 
   .then(function (storedObjects) {
-    return api.update(objectsOrIds, change)
+    if (!storedObjects) {
+      return Promise.resolve(storedObjects)
+    }
```

Comment:
I don’t think there is a case where `storedObjects` can be falsy, or is there? It’s either an object or an array